packages/cli/src/commands/alias/index.ts

```diff
@@ -38,6 +38,7 @@ const help = () => {
     -S, --scope                         Set a custom scope
     -N, --next                          Show next page of results
     -y, --yes                           Skip the confirmation prompt when removing an alias
+    --limit                             Number of results to return per page (default: 20, max: 100)
```

Comment:
Seems the convention is if the option has a value, specify a `=HINT`  with the hint being bold and underlined. The hint usual describes the thing or data type, so the hint could be `=NUM` or `=COUNT` or `=VALUE` or whatever you want.

```suggestion
    --limit${chalk.bold.underline(
    'VALUE'
  )}                             Number of results to return per page (default: 20, max: 100)
```

Reply:
<img width="1172" alt="Screen Shot 2022-10-27 at 9 57 39 AM" src="https://user-images.githubusercontent.com/74699/198352554-2e14d0a0-e49e-4f18-aba2-982d62303da1.png">

Looks much better, thank you 🙇🏾